orchestrator/careplanservice/validate_patient.go

```diff
@@ -80,8 +80,26 @@ func validatePhone(phone *string) error {
 		return errors.New("phone number is required")
 	}
 
-	if !strings.HasPrefix(*phone, "+31") {
-		return errors.New("phone number must start with +31")
+	return isDutchPhoneNumber(phone)
+}
+
+func isDutchPhoneNumber(phone *string) error {
+	normalised := strings.NewReplacer("-", "", " ", "", "(", "", ")", "").Replace(strings.TrimSpace(*phone))
+
+	normalised = strings.Map(func(r rune) rune {
```

Comment:
looks like you're doing the same normalization twice now?, the map covers every case the replacer covers?